libs/use-multi-media-devices/src/index.ts

```diff
@@ -0,0 +1,474 @@
+import { useState, useEffect, useReducer } from 'react';
+
+export type MediaDevices = {
+  cameraList: InputDeviceInfo[];
+  microphoneList: InputDeviceInfo[];
+  streams: StreamsMap;
+  initDefaultStream: () => void;
+  addStream: ({
```

Comment:
we could move the properties out from the object as individual parameters or declare the type somewhere else.
```suggestion
  addStream: ( type: StreamTypes, microphone?: InputDeviceInfo ....
```
Or 
```
type StreamOptions = {
  type: StreamThypes.....
```

Reply:
I've created a type with options